record-and-playback/core/lib/recordandplayback/generators/video.rb

```diff
@@ -54,9 +55,9 @@ def BigBlueButton.process_webcam_videos(target_dir, temp_dir, meeting_id, output
         parameters: [
           # These settings are appropriate for 640x480 medium quality, and should be tweaked for other resolutions
           # See https://developers.google.com/media/vp9/settings/vod/
-          # Increase -threads to max of 4 or increase -speed to max of 4 to speed up processing
-          %w[-c:v libvpx-vp9 -b:v 750K -minrate 375K -maxrate 1088K -crf 33 -quality good -speed 1 -g 240 -tile-columns 1 -threads 2
-             -c:a libopus -b:a 48K
+          # We use force_key_frames instead of -g to set the GOP size independent of the frame rate
+          %w[-c:v libvpx-vp9 -crf 32 -deadline realtime -cpu-used 8 -force_key_frames expr:gte(t,n_forced*10) -tile-columns 2 -tile-rows 2 -threads 4
```

Comment:
I'd suggest reverting this to maximum use of 2 threads again. Otherwise, you'll have to increase the minimum hardware requirements.
This especially causes problems for default configs with only 6 CPU cores / 12 CPU threads.

cc @antobinary @ffdixon

Reply:
regarding this, please also see the discussion of #9119

Reply:
also, why is `-cpu-used` set to 8? wouldn't that tell ffmpeg to use 50% of the available time of the cpu threads (4) at most, making it less efficient?

Reply:
Ah, I missed this. We also shouldn't be using `-deadline realtime` when doing batch processing like this. The realtime mode is designed to run as close to 1× speed as possible by lowering encoding quality when needed, but we'd prefer having constant quality that simply encodes as fast as it can at the requested settings.

Reply:
@basisbit I see #9119 is also about switching from "good" to "realtime" and changing speed from 1 or 2 to 5. I got faster encoding with ```cpu_used 8``` than ```speed 5```, while keeping same quality. This is an example I did just now for a 37 minutes deskshare video:
```
-speed 5
encoding at 68fps (real time 2m34s, cpu time 5m35s)
SSIM Y:0.975077 (16.033934) U:0.993177 (21.660456) V:0.993852 (22.112394) All:0.981223 (17.263649)
PSNR y:28.416285 u:43.825114 v:43.393513 average:30.111927 min:7.914798 max:inf

-cpu_used 8
encoding at 84fps (real time 2m4s, cpu time 4m29s)
SSIM Y:0.974545 (15.942346) U:0.992998 (21.548044) V:0.993694 (22.002233) All:0.980812 (17.169781)
PSNR y:28.374842 u:43.735763 v:43.316691 average:30.069864 min:7.914682 max:inf
```

I believe https://trac.ffmpeg.org/wiki/Encode/VP9 has some obsolete/wrong information. For example, their CPU usage formula ```(100*(16-cpu-used)/16)%``` gets lower values when cpu_used gets higher (e.g. 0% when cpu_usage is 16). Also, it says that cpu_usage is 1...16, but when you use something higher than 8, ffmpeg errors with (tested on 16.04 and 20.04): 
```
Value 16.000000 for parameter 'cpu-used' out of range [-8 - 8]
Error setting option cpu-used to value 16.
```

Regarding ```-threads 4```, we need to keep in mind that 1 thread doesn't mean 1 CPU busy 100%. In fact, vp9 codec doesn't scale very well with the number of threads. When you have 4 threads most of them are working at aroud 40%, so you're using less than 2 cores. Here is an example I did on that same file:
```
- 8 thread (actually limited by libvp9 to 4 due to input resolution and tile-rows/columns)
339.43user 13.18system 3:09.33elapsed 186%CPU (0avgtext+0avgdata 197688maxresident)k
- 4 threads
346.91user 14.32system 3:17.90elapsed 182%CPU (0avgtext+0avgdata 197816maxresident)k
- 3 threads
segmentation fault
- 2 threads
319.64user 7.71system 3:31.64elapsed 154%CPU (0avgtext+0avgdata 190108maxresident)k
- 1 thread
212.43user 1.87system 3:06.47elapsed 114%CPU (0avgtext+0avgdata 186460maxresident)k
```

However, your comment made me think to something else: the settings for x264 are not limited in threads and unlike vp9, x264 will happily use as much as possible CPU:
```
- without threads (current pull)
324.41user 21.46system 1:08.27elapsed 506%CPU (0avgtext+0avgdata 273496maxresident)k
- with threads 4
296.78user 11.51system 1:38.49elapsed 312%CPU (0avgtext+0avgdata 156868maxresident)k
```
So maybe we should add ```-threads 4``` to FFMPEG_WF_ARGS.

@kepstin I don't consider ```deadline``` as a real issue, as long as the encoder can do more than 1x (which means 5fps for deskshare or 15fps for webcams). My VMs do around 60fps (or 11,4x for deskshare, 3.8x for webcams). My guess is that anyone with vp9 encoding speed lower that 1.5x (in realtime mode), will decide not to use vp9 at all as the "good" enconding mode takes far too long.